docs/content/en/main/concepts/investigations/edge-multi-cluster.md

```diff
@@ -0,0 +1,82 @@
+  by reducing the size of the control plane to a single binary. Furthermore, KCP's Transparent Multi-cluster (TMC) has made it possible to define
+  and deploy workloads centrally and then fan them out to specific edge locations. We would 
+   like to leverage different topologies (for instance: cascading multiple instances of KCP and Kubernetes cluster) to arrive at a 
+    distributed system capable of reaching any targeted edge cluster with/without edge devices attached. Topology flexibility 
+     will require a scalable database to hold the objects referenced by each of their associated API servers. The default database for 
+      KCP and Kubernetes is etcd. We will investigate alternatives to etcd distribution and replacement databases to increase 
+       scalability without sacrificing performance and resilience and considering factors which could negatively impact cost.
+
+## Areas of investigation
+
+1. Define the high level user Edge multicluster use case
+2. Explore characteristics of most common Kube workload objects that could allow them to be placed in edge cluster locations
+3. Desired placement expression​: Need a way for one center object to express large number of desired copies​
+4. Scheduling/syncing interface​: Need something that scales to large number of destinations​
+5. Rollout control​: Client needs programmatic control of rollout, possibly including domain-specific logic​
+6. Customization: Need a way for one pattern in the center to express how to customize for all the desired destinations​
+7. Status from many destinations​: Center clients may need a way to access status from individual edge copies
+8. Status summarization​: Client needs a way to say how statuses from edge copies are processed/reduced along the way from edge to center​.
+9. Seek consensus in user communities on whether the interfaces are practical
+10. Formalize parts of the prototype into project(s) drawing on the elements above if successful!
+
+<!-- ## Use cases -->
+<!-- placeholder for future -->
+<!-- Representing feedback from a number of multi-cluster users with a diverse set of technologies in play: -->
+
+### As a user
+
+1. I can `kubectl apply` a workload that is agnostic to node placement to `kcp` and see the workload assigned to many edge location
+ resources and start running and the status summarized back to me.
+2. I can move an application (defined in 1) between two workload clusters by changing a single high level attribute
+3. As a user when I move an application (as defined in 2) no disruption of internal or external traffic is visible to my consumers
+4. As a user I can debug my application in a familiar manner regardless of cluster
+5. As a user with a stateful application by persistent volumes can move / replicate / be shared across clusters in a manner consistent
+ with my storage type (read-write-one / read-write-many).
+
```

Comment:
Few more:
- As an edge admin I can define a "template" for group of edge locations and be able to apply rule-based customization for specific edge locations.
- As an admin I can watch a summary of a group of edge locations
- As an admin I can request a detailed status  for specific edge location
- As an admin I can define roles for accessing and managing the edge locations and applications on the edge locations

Reply:
added as suggested - thank you